Makefile

```diff
@@ -56,8 +56,11 @@ website/build-local:
 .PHONY: license-check
 license-check:
 	go mod vendor
-	licensei check
-	licensei header
+	licensei cache --debug
+	licensei check --debug
+	licensei header --debug
+	rm -rf vendor/
+	git diff --exit-code
```

Comment:
This ensure that the check fails if any file is added, modified or deleted during processing checks.